.circleci/matrix.yml

```diff
@@ -1,13 +1,23 @@
 env:
+  - SOLIDUS_BRANCH=v2.0 DB=postgres
```

Comment:
Since this is our extension I do not want to add these solidus versions. They add time and noise, with little value to us.

Reply:
removed